### ISSUE TITLE
Displaying additional info in one public-facing record

### ISSUE THREAD

**Issue by kjrawson** [CollegeOfTheHolyCross/dta_sufia#96]
Hi, Steven!
I have only seen this in one record (which I just posted today), but there are some things displaying that shouldn't be there (picture attached). Thanks!

![recorderror](https://cloud.githubusercontent.com/assets/15388345/20842055/25a9b6fa-b884-11e6-8d67-420472b6c114.jpeg)

**Comment by scande3** [comment]
That's really weird! No theoretical clue how that could be occurring. I'll look at it and fix it this weekend. Owe you an update on other outstanding issues so will make time for that as well.

**Comment by kjrawson** [comment]
I thought it was strange, too. I did try to change the visibility from within the record itself, and it gave me an error. So, I went to the primary file list and did a "swap visibility" there and it showed up fine, just with the extra stuff.
Thanks for looking into this!

**Comment by scande3** [comment]
@kjrawson - This issue has been fixed. It was caused because I had commented out some of the default template includes in the code. However, when FITS (a file analysis tool that didn't like this file for some reason but isn't essential to code operation) throws an error, a part of the commented out code would activate. This code includes a closing comment tag that would end my code's attempt to not include this information and display everything after that closing comment tag.

TLDR: I removed the include from the design completely and it shouldn't occur again. Next update will be Wed based on my current schedule.

**Comment by kjrawson** [comment]
great. thanks, steven!
